packit_service/models.py

```diff
@@ -161,16 +168,10 @@ class CoprBuild(Base):
     data = Column(JSON)
 
     def set_status(self, status: str):
-        session = get_sa_session()
         self.status = status
-        session.add(self)
-        session.commit()
```

Comment:
How is this going to propagate into database?

Reply:
`autocommit=True`

every time you change a SA object, it gets automatically committed to the database; see the tests :) which means we no longer need to do `.commit()`

Reply:
Neat!